ensure-issue-authorization/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Ensure issue authorization
+description: Ensures issues are authorized and automatically closes unauthorized issues
+author: Karlo Nicholas Soriano
+inputs:
+  close-message:
+    description: Message to comment on the issue before closing
+  lock-message:
+    description: Message to comment on the issue before locking
+  lock-on-permission:
+    description: When user's permission level matches this, issue will be locked
```

Comment:
Do we need multiple? e.g. we want to lock if someone has write or admin role. Or do both roles have the write permission? No idea how it works :)

Reply:
To support that, we would need to change the parameter to receive an array of roles.

I get the user's permission level from https://developer.github.com/v3/repos/collaborators/#review-a-users-permission-level and I think they're mutually exclusive.